feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/TimetableScreen.kt

```diff
@@ -146,11 +146,14 @@ fun TimetableScreen(
                 when (uiState.timetable) {
                     is TimetableUiState.Empty -> Text("Empty")
                     is TimetableUiState.GridTimetable -> {
+                        val timetableGridUiState = requireNotNull(uiState.timetable.timetableGridUiState[selectedDay])
                         TimetableGrid(
-                            timetable = requireNotNull(uiState.timetable.timetableGridUiState[selectedDay]).timetable,
+                            timetable = timetableGridUiState.timetable,
                             timeLine = uiState.timetable.timeLine,
                             onTimetableItemClick = onTimetableItemClick,
                             selectedDay = selectedDay,
+                            isBookmarked = { id -> timetableGridUiState.timetable.bookmarks.contains(id) },
```

Comment:
[nitpick] Since we already pass the reference to `timetable`, I think we may not need to pass `isBookmarked` separately here. We could just call `timetable.bookmarks.contains(id)` inside `TimetableGrid`.

Reply:
@kitakkun 
Fixed in the following commit. 🫡 
https://github.com/DroidKaigi/conference-app-2025/pull/554/commits/1c09f6a62484305e9e6b27b978a83704a7b59134